src/jsTest/kotlin/it/krzeminski/snakeyaml/engine/kmp/internal/TargetUtils.kt

```diff
@@ -0,0 +1,5 @@
+package it.krzeminski.snakeyaml.engine.kmp.internal
+
+// returns 'true' is we are in the browser
```

Comment:
note: probably initially it was true initially but right now it returns `true` if we are not in the browser

Reply:
https://github.com/krzema12/snakeyaml-engine-kmp/pull/332/commits/76fc1145106324faff2f934812a2e2bd75aa5f67